tests/90jira/SYN-627.pl

```diff
@@ -0,0 +1,43 @@
+test "Events come down the correct room",
+   requires => [ local_user_fixture( with_events => 0 ), "can_sync" ],
+
+   check => sub {
+      my ( $user ) = @_;
+      my ( @rooms );
+
+      Future->needs_all( map {
+         matrix_create_room( $user )
+         ->on_done( sub {
+            my ( $room_id ) = @_;
+
+            push @rooms, $room_id;
+         });
+      } 0 .. 30 )
```

Comment:
0 .. 30 creates 31 instances; due to inclusive ranges. It probably doesn't matter here, but for clarity you probably want to write the more idiomatic `1 .. 30`